README.md

```diff
@@ -50,6 +50,12 @@ For entitlements, only users from those groups are selectable which have to be c
 
 ![advanced permission entitlement](screenshots/aclAdmin.png)
 
+## Configuration parameters
+
+Some settings are currently only exposed via `config/config.php`:
+
+Configures the default quota used when creating new groupfolders: `'groupfolders.quota.default' => -3,`
```

Comment:
Maybe add the meaning of special values like `-3`.

Reply:
I was searching for some general documentation about the meaning, but I couldn't find any. Do you know if we have something like that?

Reply:
From the code, it looks like the only special value is `-3`. You can also specify the unit of the value, which seem to be in bits.

Reply:
It's bytes ;)